src/common/components/select-repository-search/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './search.css';
+
+export default class SelectRepositorySearch extends Component {
```

Comment:
Something to consider - as this component has only rather simple render function it can be just a function, not a class:

```js
export default function SearchInput(props) {
  let id // ... etc
  return (
     <label> ... </label>
  )
}